src/Domain/Helper/Files.php

```diff
@@ -21,21 +22,31 @@ final class Files
      */
     public static function find(string $basedir, array $list): array
```

Comment:
Could you explain the changes here. Not sure I get them completely :)

Reply:
Yes ! This helper is used by our Insight to get a list of excluded files. 

Most of time, when we set an exclude list in `phpinsights.php`, we pass an array of files, not of directory.

So when there is only files, we create by ourself the list of SplFileInfo instead of using Finder. It's really faster. 
But if we place a directory in exclude list, we add it in path of finder and use the finder to get files.

Reply:
Ah nice, thanks for explaining!